c_binding/lsm_convert.hpp

```diff
@@ -300,4 +301,29 @@ int LSM_DLL_LOCAL values_to_uint32_array(Value & value,
 Value LSM_DLL_LOCAL uint32_array_to_value(uint32_t *uint32_array,
                                           uint32_t count);
 
+/**
+ * Converts a Value to a lsm_battery
+ * @param battery  Value representing a battery
+ * @return lsm_battery pointer, else NULL on error
+ */
+lsm_battery LSM_DLL_LOCAL *value_to_battery(Value &battery);
+
+/**
+ * Converts a lsm_battery to a value
+ * @param battery  lsm_battery to convert to value
+ * @return Value
+ */
+Value LSM_DLL_LOCAL battery_to_value(lsm_battery *battery);
+
+/**
+ * Converts a vector of battery values to an array.
+ * @param[in] battery_values       Vector of values that represents batterys
```

Comment:
Should change to "batteries" not "batterys" (see line 326). You'll need to update this throughout the comments if the misspelling exists elsewhere.